docs/usage.md

```diff
@@ -97,10 +97,10 @@ You will need to create a design file with information about the samples in your
 --input '[path to design file]'
 ```
 
-It has to be a comma-separated file with 2 columns, and a header row as shown in the examples below. The column headers must be `sample_id` and `data1`. By naming the `sample_id` rows uniquely, one can identify and simultaneously run multiple replicates and samples:
+It has to be a comma-separated file with 2 columns, and a header row as shown in the examples below. The column headers must be `sample` and `fastq`. By naming the `sample` rows uniquely, one can identify and simultaneously run multiple replicates and samples:
 
 ```bash
-sample_id,data1
+sample,fastq
 exp1_rep1,clip0001_01.fastq.gz,
```

Comment:
Remove trailing commas if the samplesheet only has two columns? Unless you decided to support PE data in which case you need another column in the header.

Reply:
[Done](https://github.com/nf-core/clipseq/pull/28/commits/c4671c230371f17eb9e693c6f575f977a47f5548)